microservice/main.tf

```diff
@@ -0,0 +1,253 @@
+locals {
+  name_prefix          = "digitalekanaler"
+  shared_config        = nonsensitive(jsondecode(data.aws_ssm_parameter.shared_config.value))
+  service_account_id   = "184465511165"
+  current_region       = data.aws_region.current.name
+  internal_domain_name = "${var.application_name}.${local.shared_config.internal_hosted_zone_name}"
+
+  all_environment_secrets = merge(
+    aws_ssm_parameter.environment_secrets,
+    aws_ssm_parameter.manual_environment_secrets,
+    data.aws_ssm_parameter.external_environment_secrets,
+  )
+}
+
+#########################################
+#                                       #
+# Fargate Task                          #
+#                                       #
+#########################################
+module "task" {
+  source             = "github.com/nsbno/terraform-aws-ecs-service?ref=0.12.1"
+  application_name   = "${local.name_prefix}-${var.application_name}"
+  vpc_id             = local.shared_config.vpc_id
+  private_subnet_ids = local.shared_config.private_subnet_ids
+  cluster_id         = local.shared_config.ecs_cluster_id
+
+  cpu    = 2048
+  memory = 4096
+
+  application_container = {
+    name     = "${local.name_prefix}-${var.application_name}"
+    image    = var.docker_image
+    port     = var.port_number
+    protocol = "HTTP"
+    cpu      = 0
+
+    environment = merge(
+      {
+        DD_ENV               = var.environment
+        DD_SERVICE           = var.application_name
+        DD_VERSION           = var.datadog_version_tag
+        DD_LOGS_INJECTION    = "true"
+        DD_TRACE_SAMPLE_RATE = "1"
+        JAVA_TOOL_OPTIONS    = "-javaagent:/application/dd-java-agent.jar -XX:FlightRecorderOptions=stackdepth=256 -Xmx1024m -Xms1024m"
```

Comment:
Her setter vi vel at jvm-en kjører med 1gb minne hardkodet gjør vi ikke?

Reply:
Det er kanskje litt dumt hehe, men det er det ticket kjører med. Hva tenker du om å dra det ut som et parameter til modulen foreløpig (for å unngå for mye endringer i ticket sitt oppsett)? Og så finne ut om dette er noe som burde være likt på tvers av tjenestene senere?